main_flow/main_flow.py

```diff
@@ -87,15 +87,21 @@ def periodically_save_cache():
 
 def steamimfy(table):
     steamedTable = copy.deepcopy(table)
-
-    for row in steamedTable:
-        steam_list_of_words_with_cache(row)
+    steam_list_of_words_with_cache(steamedTable)
     return steamedTable
 
 
-def steam_list_of_words_with_cache(row):
-    words = row[0].replace('\n', ' ').replace('\r', ' ').replace(',', ' '). \
+
+def replace_chars_in_row(row, words):
+    words += row[0].replace('\n', ' ').replace('\r', ' ').replace(',', ' '). \
         replace('.', ' ').replace(':', ' ').replace('!', ' ').replace('?', ' ').split(' ')
+    return words
```

Comment:
A little mess here - words being appended with himself? it caused memory leak!

Reply:
fixed, now only rows are sent to the method and they are appended to "words"